dvc/output/base.py

```diff
@@ -76,6 +76,7 @@ class BaseOutput:
     PARAM_PLOT_HEADER = "header"
     PARAM_PERSIST = "persist"
     PARAM_DESC = "desc"
+    PARAM_IS_USER_EXECUTABLE = "is_user_executable"
```

Comment:
I guess we could even use `exec: true/false`, as it is specified for a particular output and it doesn't seem like we will have another use for `exec` here in the future. Could also use `isexec`. The current `is_user_executable` seems a bit long. WDYT?

Reply:
Also, `is` is very much unnecessary. `exec` or `executable` is fine imo.

Reply:
@efiop @skshetry good suggestion about a shorter variable name. It fits the style of this project well. I hesitated a lot, probably too much, between `exec` and `isexec` and decided to go with `exec`. Unfortunately, as a result, I've got a `pylint` error:

```
W0622: Redefining built-in 'exec' (redefined-builtin)
```

so I've started using `isexec` instead. Please let me know if you have a stronger opinion about using `executable` instead.

Reply:
> good suggestion about a shorter variable name.

The suggestion was regarding the keyword in the dvc.yaml/dvcfile, not the variable name. I have no strong opinion towards either `exec` or `isexec`, but the decision should not be based on that pylint complained.

Reply:
I think that `isexec` seems better than `exec` as the latter sounds like "execute this output: true" which is confusing.